tools/differ/helper.py

```diff
@@ -0,0 +1,88 @@
+import glob
+import os
+import pandas as pd
+import re
+
+from google.cloud.storage import Client
+
+
+def load_mcf_file(file: str) -> pd.DataFrame:
+    """ Reads an MCF text file and returns it as a dataframe."""
+    mcf_file = open(file, 'r', encoding='utf-8')
+    mcf_contents = mcf_file.read()
+    mcf_file.close()
+    # nodes separated by a blank line
+    mcf_nodes_text = mcf_contents.split('\n\n')
+    # lines seprated as property: constraint
+    mcf_line = re.compile(r'^(\w+): (.*)$')
```

Comment:
there can be spaces around `:`
for example: `Node : dcid:TestNode` is valid.

Reply:
Updated the expression to  mcf_line = re.compile(r'^(\w+)\s*:(.*)$')